### ISSUE TITLE
Cant find Complex Softmax which takes complex input and output complex output

### ISSUE THREAD

**Issue by Karn3003** [NEGU93/cvnn#38]
Hi,
I am unable to find the complex softmax activation function.

**Comment by NEGU93** [comment]
[Here](https://complex-valued-neural-networks.readthedocs.io/en/latest/activations/real_output.html#softmax-based) you have ;)

**Comment by NEGU93** [comment]
Yes, I forgot to add the function to the docs. It is now done.
Please check [`cart_softmax`](https://complex-valued-neural-networks.readthedocs.io/en/latest/activations/types.html#cart_softmax). Beware that your loss function should take complex inputs. Like `ComplexAverageCrossEntropy`
